src/main/resources/hudson/plugins/copyartifact/BuildSelectorParameter/config.jelly

```diff
@@ -22,12 +22,12 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:ca="/hudson/plugins/copyartifact">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
   <f:entry title="${%Name}" field="name" help="/help/parameter/name.html">
     <f:textbox/>
   </f:entry>
-  <ca:selectorList currentSelector="${instance.defaultSelector}"
-      name="defaultSelector" title="${%Default Selector}" omit="ParameterizedBuildSelector"/>
+  <f:dropdownDescriptorSelector field="defaultSelector"  help="/plugin/copyartifact/help-buildSelector.html"
+      title="${%Default Selector}" descriptors="${descriptor.availableBuildSelectorList}" />
```

Comment:
I considered it again, and found that it would result creating three copies of help-selector.html.
I'll add a commit to revert the move of help-defaultSelector.html in #62.